### PR TITLE
use new valid composer syntax

### PR DESCRIPTION
**Pull request by OskarStark**
### Subject

The build was failing, due to a new composer lint check

### Example

https://travis-ci.org/sonata-project/SonataDoctrineORMAdminBundle/jobs/134351567

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,13 +24,13 @@
         "symfony/console": "^2.3 || ^3.0",
         "symfony/doctrine-bridge": "^2.2 || ^3.0",
         "symfony/framework-bundle": "^2.2 || ^3.0",
-        "symfony/security-acl": "^2.2 || ^3.0",
+        "symfony/security-acl": "^2.2 || ^3.0^",
```
Comment (Soullivaneuh) [inline composer.json]: `^3.0^` ? And `symfony/security-acl` cans fly? :trollface:

File: composer.json [hunk 2]
```diff
@@ -24,13 +24,13 @@
...
         "sonata-project/exporter": "^1.3.1",
         "sonata-project/admin-bundle": "^3.0",
         "sonata-project/core-bundle": "^3.0"
     },
     "require-dev": {
-        "simplethings/entity-audit-bundle": "~0.1",
+        "simplethings/entity-audit-bundle": "^0.1",
```
Comment (Soullivaneuh) [inline composer.json]: Could you try `0.1 - 0.9` and check if lowest and highest versions are correctly downloaded?

https://packagist.org/packages/simplethings/entity-audit-bundle#0.9.1

File: composer.json (hunk 1 above)
Comment (OskarStark) [inline composer.json]: haha

File: composer.json (hunk 2 above)
Comment (OskarStark) [inline composer.json]: > Could you try 0.1 - 0.9 and check

do you mean `>= ^0.1 || <= ^0.9` ?

File: composer.json (hunk 1 above)
Comment (OskarStark) [inline composer.json]: fixed

File: composer.json (hunk 2 above)
Comment (Soullivaneuh) [inline composer.json]: `>= ^0.1 || <= ^0.9` is invalid. I think you meant `>= 0.1 || <= 0.9`.

Hyphen does the same AFAIK: `0.1 - 0.9`

File: composer.json [hunk 3]
```diff
@@ -31,7 +31,7 @@
     },
     "require-dev": {
         "knplabs/knp-menu-bundle": "^2.1.1",
-        "simplethings/entity-audit-bundle": "~0.1",
+        "simplethings/entity-audit-bundle": "^0.1",
```
Comment (Soullivaneuh) [inline composer.json]: `0.1 - 0.9`

I just tested, it works.

File: composer.json (hunk 2 above)
Comment (OskarStark) [inline composer.json]: yes thats what i meant.

shall i change it to this?

Comment (Soullivaneuh) [inline composer.json]: Yes, use `0.1 - 0.9`.

Comment (OskarStark) [inline composer.json]: done

File: composer.json (hunk 3 above)
Comment (OskarStark) [inline composer.json]: done

**Pull request merged**